components/collapse/Collapse.tsx

```diff
@@ -163,6 +164,7 @@ const Collapse = React.forwardRef<HTMLDivElement, CollapseProps>((props, ref) =>
       prefixCls={prefixCls}
       className={collapseClassName}
       style={{ ...collapse?.style, ...style }}
+      {...otherProps}
```

Comment:
https://github.com/ant-design/ant-design/blob/31ff625d8f9138dcb82079223ac7c255358b9d91/components/radio/group.tsx#L110

Reply:
Looks like I overlooked that we already doing spread here https://github.com/ant-design/ant-design/pull/47897/files/57f8be7c57c325c8916e08399a8ec33dd9845349#diff-3541251bb31318316b91e1f6710b646bd92edaff4318ca790c6401bea6962fc9R162

And it could be not a problem in Antd. Sorry for disturbing